packages/pageheader/README.md

```diff
@@ -0,0 +1,19 @@
+# PageHeader Name
+
+PageHeader documentation
+
+Example Implementation
+
+```JSX
```

Comment:
What's the thinking behind adding example implementation in the README? Storybook can do that automatically.

Reply:
This is also a left over from before. Storybook had problems with having objects as Children and I thought it would be helpful to have the component used in code at least once, will remove the documentation here.